app/views/users/user_homepage.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Welcome, <%= @user.username %> - These are your tweets!</h1>
+
+  <% if !@user.tweets.empty? %>
+    <% @user.tweets.each do |tweet| %>
```

Comment:
In the MVC design pattern we should avoid database calls in the view. You would be better served setting this to an instance variable in the get `'/users/:id'` do method
```ruby
@tweets = current_user.tweets
```